### PR TITLE
Azure pipelines

### LINKED ISSUE #213: [CICD] Migrate from Travis to Azure Pipelines
Currently, we are using `Travis` as our CI/CD tool. This issue is to discuss migrating to `Azure Pipelines`.

Comments:
* Azure Pipelines are used in our `conda-forge` [`scikit-multiflow-feedstock`](https://github.com/conda-forge/scikit-multiflow-feedstock) to create artifacts for multiple environments.
  * Potentially we can use Azure Pipelines to deploy artifacts to `PyPI`
  * Note: Is this possible in Travis?
* Azure Pipelines provides unlimited CI/CD minutes and 10 parallel jobs to every open source project for free.

### PR DESCRIPTION
**Pull request by jacobmontiel**
Changes proposed in this pull request:

* Add configuration for azure-pipelines
* Minor test updates due to system differences between Travis and Azure agents
* Fixes #213 

---

Checklist

- [x] Code complies with PEP-8 and is consistent with the framework.
- [x] Code is properly documented.
- [x] Tests are included for new functionality or updated accordingly.
- [x] Travis CI build passes with no errors.
- [x] Test Coverage is maintained (threshold is -0.2%).
- [x] Files changed (update, add, delete) are in the PR's scope (no extra files are included).

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        pip install --only-binary=numpy,scipy numpy scipy
+        pip install cython
+        pip install .
+      displayName: 'Installing scikit-multiflow'
+
+    - script: |
+        pip install pytest
+        pip install pytest-cov
+        pytest tests --showlocals -v --cov=skmultiflow
+      displayName: 'Running tests'
```
Comment (garawalid) [inline azure-pipelines.yml]: I'd create a `requirements-dev.txt` for `pytest`, `wheel`, and others. Then I install both `requirements.txt` and `requirements-dev.txt` in this stage.  
So changes will be made only in one place.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# Starter pipeline
```
Comment (garawalid) [inline azure-pipelines.yml]: nit: unnecessary comments

**Comment by smastelini** [comment]
Thanks for your comments and suggestions @garawalid!

File: azure-pipelines.yml (hunk 1 above)
Comment (jacobmontiel) [reply azure-pipelines.yml]: I like the simplicity of this approach, thanks for the suggestion. I will try it.

Comment (jacobmontiel) [reply azure-pipelines.yml]: Ended up with a hybrid approach. The following steps must be run individually:

```
python -m pip install --upgrade pip
python -m pip install setuptool
```

Comment (garawalid) [reply azure-pipelines.yml]: It's clean. Thanks @jacobmontiel

**Pull request merged**